src/components/application_manager/src/rpc_service_impl.cc

```diff
@@ -390,7 +390,10 @@ bool RPCServiceImpl::ManageHMICommand(const commands::MessageSharedPtr message,
 
   if (command->Init()) {
     command->Run();
-    if (kResponse == message_type) {
+    if (helpers::Compare<int32_t, helpers::EQ, helpers::ONE>(
```

Comment:
resumption related also

Reply:
I would likely say that these changes are related to the Persisting HMI Capabilities feature and we need them to check that SDL postpones StartSessionAck until all vehicle data is available. 
1. Changes in get_capabilities_response.cc / get_language_response.cc etc. are needed because sometimes after postponing of StartSessionAck mobile gets default values, not the ones sent by HMI, because of the timing issue and it has been fixed in https://github.com/smartdevicelink/sdl_core/pull/3616/commits/e70cd8474defd581589bf46e0a33633bf746fba2
2. And the changes in rpc_service are needed because if HMI responds with an error to BC.GetSystemInfo - SDL wouldn't terminate the request and the `hmi_cooperating_`  flag wouldn't be set to true until BC.GetSystemInfo is expired by timeout and StartSessionAck all this time will be postponed. And it has been fixed in https://github.com/smartdevicelink/sdl_core/pull/3616/commits/dc3af681f0fa52c033deffcf916c79e243d57e8f